Assets/MixedRealityToolkit-Preview/LightEstimation-Examples/EditorGyro/EditorGyro.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+#if UNITY_EDITOR_WIN
+using System.Runtime.InteropServices;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit.LightEstimation
+{
+	public static class EditorGyro
+	{
+		#if UNITY_EDITOR_WIN
+		[DllImport("GyroDLL", EntryPoint = "GyroInitialize")]
```

Comment:
At the very least, I'll link to the source in the code! Would definitely like to know what our stance is though. As a note, I did put these in the Example folder, so it wasn't as important to clutter things up with the DLL source if it wasn't necessary.

https://github.com/maluoi/UnityEditorGyro

Reply:
Another note, the EditorGyro DLL is editor only, and does not get compiled into any distributables.

Reply:
Currently, for experimental features, there is no requirement for DLLs to be open source. They should be signed to enable customers to trust the lineage. They must be licensable under the MIT license.

We are investigating how best to handle DLLs that ship with MRTK going forward. It is easy to envision valid instances where official and extension components may rely upon a closed source binary. One decision that has been made is that all DLLs that ship with official MRTK components _must_ be signed.

Reply:
After talking with people, signing DLLs seems like a complicated task right now. Since it's not an essential feature, I've removed the DLLs, and added instructions for people to put them back if desired!

I'll still be chasing down how to get this signed, but I don't really want this to be a blocker at the moment!